src/reducers.js

```diff
@@ -0,0 +1,182 @@
+import { predefinedGames } from './utils/predefined-games';
+import bots from './bots';
+import { interactiveCellTypes } from './utils/constants';
+import { CLICK_CELL, CHANGE_GAME_INDEX, RESET_GAME, CHANGE_BOT, BOT_RUNNING } from './actions';
+import { combineReducers } from 'redux';
+
+// Initial game index to load.
+const INITIAL_GAME_INDEX = 0;
```

Comment:
The typical pattern for default state is by passing in your initial state to `createStore` when you create your Redux store. So instead of having this constant here in your reducer, you should instead put it in your `App.js` file and pass your initial state in as the second argument to `createStore`. Something like this:

```javascript
// ...
createStore(reducers, {key1: val1, key2: val2}); 
// ...
```